domains/duypt14.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ptduy14",
+        "email": "phantanduy14@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://portfolio-ptduy14s-projects.vercel.app/"
```

CNAME can't contain protocol or slashes
```suggestion
        "CNAME": "portfolio-ptduy14s-projects.vercel.app"
```

---

many thanks